msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md

```diff
@@ -1,246 +1,348 @@
 ---
-title: App submission tips and frequently failed cases
-description: describes tips for a successful Teams store submission and common reasons submissions fail
+title: Microsoft Teams store validation guidelines
+description: Describes the guidelines every app submitted to the Teams store (AppSource) must follow.
+author: heath-hamilton
+ms.author: surbhigupta
 ms.topic: reference
-ms.author: lajanuar
-keywords: app submission tips frequently failed cases validation guidelines
 ---
-# Tips for a successful Microsoft Teams app submission
+# Microsoft Teams store validation guidelines
 
-This article addresses common reasons submitted apps fail validation. While it's not intended to be an exhaustive list of all potential issues with your app, following this guide will increase the likelihood that your app submission will pass the first time. See [Commercial marketplace certification policies](/legal/marketplace/certification-policies) for an extensive list of validation policies.
+Reviewing these guidelines can help you avoid issues that frequently cause apps to fail Microsoft Teams store and AppSource submission. These guidelines complement the Microsoft [commercial marketplace certification policies](https://docs.microsoft.com/legal/marketplace/certification-policies) and will be updated frequently to reflect new Teams capabilities, user feedback, and changes to business rules.
 
->[!NOTE]
->**[Section 1140](/legal/marketplace/certification-policies#1140-teams)** is specific to Microsoft Teams and **[sub-section 1140.4](https://docs.microsoft.com/legal/marketplace/certification-policies#11404-functionality)** addresses functionality requirements for Teams apps.
+> [!TIP]
+> Some guidelines may not be applicable to your app. For example, if your app doesn't include a bot, you can ignore bot-related guidelines.
+
+## 1.0 Value proposition
+
+### 1.1 App name
+
+An app's name plays a critical role in how users discover it in the store. Remember the following about app names:
+
+* The name must include terms relevant to your users.
+* It must not copy the name of an existing app in the store or other offer in the commercial marketplace.
+* Common nouns must be prefixed or suffixed with the publisher’s name (for example, “XYZ Tasks” rather than “Tasks”)
```

Comment:
•	Apps should not use Teams default module names with in their app names like "Chat" , "Contacts", "Calendar", "Calls", "Files", "Activity", "Teams", Apps", "Help".
•	Apps should not use the Teams feature specific keywords like - Start Meeting, Make call, Upload file, Start chat, etc - these keywords will be reserved for Teams and Microsoft products first. For every other 3P app, we will mandate that they include their app name before the CTA: Eg: Start a Zoom Meeting, Upload file to Dropbox, Start Freshdesk chat etc.

Reply:
@ramendhe For the second bullet, I am adding this in a new section (1.4) since this is not just related to the app name. Let me know if you think another approach is better.

Reply:
@heath-hamilton, Please reword 1.4 as this applies to Teams specific features key works only. Not all features require app name to be included - 

Apps should not use the Teams feature specific keywords like - Start Meeting, Make call, Upload file, Start chat, etc - these keywords are reserved for Teams and Microsoft products first. For 3P app, we expect feature to include app name before the CTA: Eg: Start a Contoso Meeting, Upload file to Dropbox, Start Contoso chat etc.